app/database/db.go

```diff
@@ -314,6 +314,17 @@ func (conn *DB) Count(dest interface{}) *DB {
 	return newDB(conn.db.Count(dest))
 }
 
+// Pluck used to query single column from a model as a map
```

Comment:
I know that is from gorm but I do not find this sentence very clear.

Reply:
fixed